packages/graphql/tests/integration/issues/1685.int.test.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { graphql, GraphQLSchema } from "graphql";
+import type { Driver } from "neo4j-driver";
+import Neo4j from "../neo4j";
+import { Neo4jGraphQL } from "../../../src";
+import { generateUniqueType } from "../../utils/graphql-types";
+
+describe("https://github.com/neo4j/graphql/issues/1685", () => {
+    const productionType = generateUniqueType("Production");
+    const movieType = generateUniqueType("Movie");
+    const genreType = generateUniqueType("Genre");
+
+    let schema: GraphQLSchema;
+    let neo4j: Neo4j;
+    let driver: Driver;
+
+    async function graphqlQuery(query: string) {
+        return graphql({
+            schema,
+            source: query,
+            contextValue: neo4j.getContextValues(),
+        });
+    }
+
+    beforeAll(async () => {
+        neo4j = new Neo4j();
+        driver = await neo4j.getDriver();
+
+        const typeDefs = `
+            interface ${productionType.name} {
+                id: ID
+                title: String
+            }
+            
+            type ${movieType.name} implements ${productionType.name} {
+                id: ID
+                title: String
+                ${genreType.plural}: [${genreType.name}!]! @relationship(type: "HAS_GENRE", direction: OUT)
```

Comment:
You don't need to use unique types for the names of the parameters, as they are just the aliases of the responses and not used in the database, it usually leads to easier to write and read tests:

```suggestion
               genres: [${genreType.name}!]! @relationship(type: "HAS_GENRE", direction: OUT)
```